packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewerContainer.js

```diff
@@ -8,10 +8,14 @@ import locationValidator from '../../helpers/locationValidator'
 class SingleImageViewerContainer extends React.Component {
   constructor () {
     super()
+    this.imageViewer = React.createRef()
+    this.onLoad = this.onLoad.bind(this)
     this.onError = this.onError.bind(this)
     this.state = {
-      height: null,
-      width: null,
+      clientHeight: 0,
+      clientWidth: 0,
+      naturalHeight: null,
+      naturalWidth: null,
```

Comment:
Should the natural sizes also be 0? Just to be consistent with type with the client sizes

Reply:
Sure. They were `null` in the original code but that is inconsistent with the model type.